config/admins.txt

```diff
@@ -142,3 +142,4 @@ Skoglol = Game Master
 Time-Green = Game Master
 StyleMistake = Game Master
 actioninja = Game Master
+Wineallwine = Game Master
```

Comment:
lol my bad

Reply:
nobody would notice ;)

Reply:
its not an issue to do this, it's pretty common to add your name to this as it's overriden by the config anyways.

Reply:
If you don't point it out I'd let it slide